Dmr2dds

ndp-opendap opened the pull request.

jgallagher59701 commented:
Woo Hoo.

Merge it?

What types found in a DMR don't translate? I think if it's only Opaque and Enum, we can really use this. Of course, 64-bit ints are going to be an issue... Still, it's probably very useful and we can tinker with fixes and additions on branches.

ndp-opendap commented:
OK At this point I think all of the goals have been accomplished. There may be work downstream as we start to exercise the _transform_to_dap2()_ implementation, but we'll work that out as we go. For now I have:
* repaired a bunch of _transform_to_dap4()_ implementation issues including refactoring the method so it's use can be regularized.
* Added a complete _transform_to_dap2()_ implementation with tests.
* Fixed issues in _transfer_attributes()
All the tests pass on OS-X, TravisCI, and CentOS-6.
Note that to merge I'll make a total of 4 pull requests, this one plus _bes_, _hdf4_handler_, _hdf5_handler_.

Time. To. Merge.
